docs/src/examples/components/Button/Variations/index.tsx

```diff
@@ -19,6 +19,11 @@ const Variations = () => (
       description="A button can be circular and formatted to show different levels of emphasis."
       examplePath="components/Button/Variations/ButtonExampleEmphasisCircular"
     />
+    <ComponentExample
+      title="Text"
+      description="A button can be shown in form of a text to indicate some less-pronounced actions"
```

Comment:
nit's nit: just for the sake of consistency, lets put dot at the end of description :)

Reply:
This is not a nit, is valid thing to be fixed as this is shown in our docs page :)